docs/book/v5/core-features/account-anonymization.md

```diff
@@ -0,0 +1,32 @@
+# Account anonymization
+
+According to the GDPR, companies that record personal data from EU citizens must delete said data if its owner requests its deletion.
+An alternative is to anonymize the data, according to [this article](https://commission.europa.eu/law/law-topic/data-protection/reform/rules-business-and-organisations/dealing-citizens/do-we-always-have-delete-personal-data-if-person-asks_en).
+
+## What is 'personal data'?
+
+According to [this article](https://commission.europa.eu/law/law-topic/data-protection/reform/what-personal-data_en), personal data is:
+
+- A name and surname.
+- A home address.
+- An email address such as name.surname@company.com.
+- An identification card number.
+- Location data (for example the location data function on a mobile phone).
+- An Internet Protocol (IP) address.
+- A cookie ID.
+- The advertising identifier of your phone.
+- Data held by a hospital or doctor, which could be a symbol that uniquely identifies a person.
+
```

Comment:
So the user's phone number does not count as personal data? That's odd...

Reply:
i'll add an entry for the phone number explicitly